content/ember-5-0-released.md

```diff
@@ -0,0 +1,330 @@
+- [The Ember Project](https://blog.emberjs.com/ember-project-at-2-0/)
+- [Ember LTS Releases](https://blog.emberjs.com/announcing-embers-first-lts/)
+
+## Beta cycle
+
+This release kicks off the 5.1 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+
+## How to upgrade to Ember 5
+
+The most common approach for upgrading to 5.0 is to upgrade your app to the last
+version of Ember 4, which is 4.12, resolve all deprecation warnings, and then upgrade to 5.0. If your app or addon runs with no deprecations in the latest release of 4.12,
+you should be able to upgrade to Ember 5 with no additional changes. Step by step directions are below.
+
+If your app is at a much earlier version in the 4.x series, we
+recommend upgrading in steps across the LTS versions: 4.4, 4.8, and then 4.12.
+
+Follow these steps in order:
+
+1. Upgrade your project to the latest patch version of Ember 4.12.
+   Many developers can do this by running
+   `npx ember-cli-update --to 4.12`. The
+   [general Ember upgrade guide](https://cli.emberjs.com/release/basic-use/upgrading/#upgradinganemberappitself)
+   has more details about how to upgrade your Ember app/addon.
+2. Make sure your app builds successfully.
+3. Resolve all deprecation warnings. These Deprecated APIs are removed in Ember 5.0. You may need to upgrade some of your addon dependencies if they are using deprecated APIs.
+See the [Ember Deprecation Guide](https://deprecations.emberjs.com/)
+for more details about specific deprecations and how to resolve them.
+Applications that need to upgrade through several versions may want to
+consider isolating individual deprecations by using
+[ember-cli-deprecation-workflow](https://github.com/mixonic/ember-cli-deprecation-workflow).
+    - *Note:* If you see a deprecation for `@ember/string`, please review the
+[deprecation guide for importing from @ember/string without @ember/string as a dependency](https://deprecations.emberjs.com/v4.x#toc_ember-string-add-package) for instructions.
+There is a known bug in the implementation of this deprecation.
+4. Make sure your app builds successfully and your test suite passes with no
+   deprecations.
+5. Upgrade your app to Ember 5.0. Again,
+   many developers can do this by running
+   `npx ember-cli-update --to 5.0`. Refer to the
+   [general Ember upgrade guide](https://cli.emberjs.com/release/basic-use/upgrading/#upgradinganemberappitself)
+   for more details about how to upgrade your Ember app/addon.
+
+For help or advice along the way, visit [Ember's forums or chat groups](https://emberjs.com/community/).
+
+<!-- TODO - do we need to mention anything here about htmlbars, fastboot, embroider, or ember-auto-import? -->
```

Comment:
Embroider and auto-import support back to 3.28, so i don't think so from that perspective

Reply:
ok, thank you!